frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart

```diff
@@ -15,23 +16,30 @@ class GridAddRowButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final color = Theme.of(context).brightness == Brightness.light
+        ? const Color(0xFF171717).withOpacity(0.4)
+        : const Color(0xFFFFFFFF).withOpacity(0.4);
+    final borderColor = Theme.of(context).isLightMode
+        ? const Color(0xFFEDEDEE)
```

Comment:
the value of `theme.lightBorderColor` is `Color(0xFFEDEDEE)`

Reply:
Oh nice. Done